atlas/packages/app/src/api/queries/channels.ts

```diff
@@ -18,3 +20,22 @@ export const GET_NEWEST_CHANNELS = gql`
   }
   ${channelFieldsFragment}
 `
+
+export const GET_CHANNEL = gql`
+  query GetChannel($id: ID!) {
+    channel(id: $id) {
+      ...ChannelFields
+    }
+  }
+  ${channelFieldsFragment}
+`
+export const GET_CHANNEL_VIDEOS = gql`
```

Comment:
Some specific reason to run this as a separate query instead of just adding `videos` field to `GET_CHANNEL `?